api/src/models/report.ts

```diff
@@ -0,0 +1,21 @@
+import mongoose from 'mongoose';
+
+const reportSchema = new mongoose.Schema({
+  reviewId: {
```

Comment:
it seems like it is capitalized everywhere else and case matters
```suggestion
  reviewID: {
```